src/pages/documentation/resources/upgrade-platform-sdks/comparison.md

```diff
@@ -35,14 +35,14 @@ The following tables provide information about the differences between the new E
 
 | Functionality | 4x SDK | Experience Platform SDK |
 |---|---|---|
-| Postbacks - Get/POST URL requests | Yes |	Yes - [Signals](../mobile-core/signal/index.md) extension |
-| Postbacks - PII Get/POST URL requests | Yes | Yes - [Signals](../mobile-core/signal/index.md) extension |
-| Postbacks - Open app deeplink | Yes | Yes - [Signals](../mobile-core/signal/index.md) extension |
-| Push Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/) extension |
-| In-app Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/) extension |
-| Marketing/Acquisition Links | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/) extension |
-| Geo location and beacon tracking | Yes | Yes - Use [Places extension](../places/index.md) |
-| Geo points-of-interest management | Yes | Yes - Use [Places extension](../places/index.md) |
+| Postbacks - Get/POST URL requests | Yes |	Yes - [Signals](../../mobile-core/signal/index.md) extension |
+| Postbacks - PII Get/POST URL requests | Yes | Yes - [Signals](../../mobile-core/signal/index.md) extension |
+| Postbacks - Open app deeplink | Yes | Yes - [Signals](../../mobile-core/signal/index.md) extension |
+| Push Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/) extension (deprecated) |
+| In-app Messaging | Yes | Yes - [Mobile Services](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-analytics-mobile-services/) extension (deprecated) |
```

Comment:
Why are we linking deprecated Analytics Mobile Services Push & InApp here? For latest, I think we should just mention that the functionality is supported with ACC (push only), ACS or AJO extensions.

Reply:
same here, created a dedicated task for this https://github.com/AdobeDocs/aep-mobile-sdkdocs/issues/384